src/microbe_stage/PatchMap.cs

```diff
@@ -207,7 +207,10 @@ public void UpdateGlobalPopulations()
     /// <summary>
     ///   Removes species from patches where their population is &lt;= 0
     /// </summary>
-    public void RemoveExtinctSpecies(bool playerCantGoExtinct = false)
+    /// <returns>
+    ///   The extinct creatures
+    /// </returns>
```

Comment:
I think you've broken the xml comment here.

Reply:
No, i didn't, git got confused because I modified the return value and added the `<returns>` description